builds/misc/templates/base-image-publish/base-image-build-and-publish.yaml

```diff
@@ -1,31 +1,67 @@
 steps:
 - pwsh: |
-    # Unfortunately there is no way to create a variable name (as a string) in pwsh, pass that to DevOps pipeline, and have it evaluates the value of the variable
-    # Since we know the variable names, we can hard-code variables name, and access them this way.
-    # This list must have the same version ordering as base-image-config.yaml
-    $versionTags = New-Object System.Collections.ArrayList
-    $versionTags.AddRange((
-      "$(edgeAgentTagVersion)",
-      "$(edgeHubTagVersion)",
-      "$(moduleTagVersion)",
-      "$(fullModuleTagVersion)"
-    ));
+    # Config the base image publication
+    $config = New-Object System.Collections.ArrayList
+    # Tuple format:  (ModuleRepoPath,                           BaseImageName,                   ReturningVstsVar)
+    $config.AddRange((
+      [Tuple]::Create("edge-agent",                             "azureiotedge-agent-base",       "edgeAgentTag"),
+      [Tuple]::Create("edge-hub",                               "azureiotedge-hub-base",         "edgeHubTag"),
+      [Tuple]::Create("edge-modules/SimulatedTemperatureSensor","azureiotedge-module-base",      "moduleTag"),
+      [Tuple]::Create("test/modules/TestAnalyzer",              "azureiotedge-module-base-full", "fullModuleTag")
+      ));
 
-    cd $(Build.SourcesDirectory)
-    $scriptPath = "$(Build.SourcesDirectory)/scripts/linux/createArmBase.sh"
-    $config = '$(serialize_config)' | ConvertFrom-Json
+    # Download librocksdb from blob storage
+    $artifactLocale = "$(Build.SourcesDirectory)/librocksdb.so"
+    Invoke-WebRequest -Uri $("https://edgebuild.blob.core.windows.net/librocksdb/librocksdb.so.$(arch)") -OutFile $artifactLocale
 
-    if ($config.count -ne $versionTags.count)
+    # Get all the tag for publication
+    Write-Output "`n`nBase Image Info"
+    foreach ($eachConfig in $config)
     {
-      echo '$config and $versionTags must have the same count.'
-      exit -1;
+      # Fetch tags & version from each module Dockerfiles {../$(arch)/base/Dockerfile, ../$(arch)/Dockerfile}
+      # Steps: 
+      #  1. Go to a respective Dockerfile as defined in base-image-config.yaml for an underlying base image
```

Comment:
Nit: There are a couple places in comments where you still refer to the deleted yaml files...